ringling/projects.py

```diff
@@ -0,0 +1,73 @@
+"""
+Copyright 2023 MSOE DISE Project
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import json
+import requests
+
+def get_url():
+    """
+    Get the URL for interacting with projects
+    :return: The project URL
+    """
+    return "http://localhost:8888/v1/projects"
+
+
+def create_project(project_name):
+    """
+    Create a project on the Ringling service
+    :param project_name: The name of the project to create
+    :return: The response from the service
+    """
+    obj = {"project_name": project_name}
+    try:
+        response = requests.post(get_url(),
+                                 json=obj)
+        if response.status_code == 201:
```

Comment:
What if the response isn't 201?  E.g., what if it can't create the project because there already is a project with that name?  You need to handle other cases.  The hundreds place will tell you if it's successful or not.  See [here](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status)

Reply:
It looks like I can create multiple projects with the same name, and they all give 201 as well.
The REST API docs says that it can only return a 201 or a 400 if project name doesn't exist, but the client won't allow the request to be sent without a project name. I can still make it handle other cases though if that would be helpful / best practice.